Gemfile.lock

```diff
@@ -495,6 +499,13 @@ GEM
     shellany (0.0.1)
     shoulda-matchers (3.0.0)
       activesupport (>= 4.0.0)
+    sidekiq (4.2.2)
```

Comment:
I think we do need them, it uses the same trick we have for the `pg` them. The gem is added and locked but only installed by `bundle install` when the ENV config is matching.

Without it `bundle install --deployment` would complain that the lockfile needs to be modified, another benefit is that we know that none of the gems have version conflicts.